build/common.ps1

```diff
@@ -189,12 +190,34 @@ Function Install-NuGet {
     & $NuGetExe locals all -list -verbosity detailed
 }
 
+Function Get-MsBuildPath{
+    [CmdletBinding()]
+    param()
+
+    if (-Not (Test-Path $VsWhereExe))
+    {
+        throw "Expected VsWhere.exe at '$VsWhereExe' but it was not present. This is needed to locate MSBuild.exe"
+    }
+
+    $path = & $VsWhereExe -latest -products * -requires Microsoft.Component.MSBuild -property installationPath
+    if ($path) {
+        $path = join-path $path 'MSBuild\15.0\Bin\MSBuild.exe'
```

Comment:
Since MSBuild is moving to `current` folder going forward so we should update this code accordingly as well. Look at https://github.com/NuGet/NuGet.Client/pull/2487

Reply:
@jainaashish 

I think that change belongs in a different PR. 

VSWhere ships everywhere, so the fallback will likely never be used.

Reply:
its about the getting msbuild.exe path not vswhere and for now it will use fallback (like `msbuild\15.0\bin...) instead of `msbuild\current\bin...

Reply:
oh my bad, didn't look at the change again. 

I still think the point stands. 
We need to test for both and they can/should be done in a different PR.

Reply:
Actually, we didn't start shipping vswhere.exe in the Visual Studio Installer until ~Visual Studio 2017 version 15.2, so there is a small window where it might not be available on some customers' machines. I recommend keeping the fallback.

Reply:
@heaths - Are there lots of 15.0's out there? The window may be larger with the announcement of 15.9 becoming SP1 and everything between it and 15.0 becoming unsupported.

Reply:
I can't give exact numbers, but enough you might still want to consider it.

And, for the record, 15.9 is not becoming "SP1" - just the long-term servicing (LTS) baseline.